testing/tests/window.lua

```diff
@@ -0,0 +1,336 @@
+-- love.window
+
+
+-- love.window.close
+-- @NOTE closing window should cause graphics to no longer be active
+love.test.window.close = function(test)
+  love.window.close()
+  local active = false
+  if love.graphics ~= nil then active = love.graphics.isActive() end
+  test:assertEquals(false, active, 'check window active')
+  love.window.setMode(256, 256) -- reset 
+end
+
+
+-- love.window.fromPixels
+-- @NOTE dependent on the DPI value returned I think
+love.test.window.fromPixels = function(test)
+  local dpi = love.window.getDPIScale()
+  local pixels = love.window.fromPixels(100)
+  test:assertEquals(100/dpi, pixels, 'check dpi ratio')
+end
+
+
+-- love.window.getDPIScale
+-- @NOTE i dont think there's a clever way to check this 
+love.test.window.getDPIScale = function(test)
+  test:assertNotEquals(nil, test, 'check not nil')
+end
+
+
+-- love.window.getDesktopDimensions
+-- @NOTE dependent on hardware so best can do is not nil
+love.test.window.getDesktopDimensions = function(test)
+  local w, h = love.window.getDesktopDimensions()
+  test:assertNotEquals(nil, w, 'check not nil')
```

Comment:
It might be nice to have a helper function like `assertNotNil` so we don't have to write or copy/paste the 'check not nil' string everywhere